### DISCUSSION TITLE
Overstyring av sperring

### DISCUSSION THREAD

**Discussion by ChristinGorman** [NorskHelsenett/Tillitsrammeverk#55] [General]
Pasienter har mulighet til å sperre innsyn i sine helseopplysninger på forskjellige måter.  For helsepersonell i Kjernejournal vil dette føre til at de kan få opp en liste med f.eks legemidler eller prøvesvar, men i tillegg få et varsel om at det finnes sperrede dokumenter. Da kan bruker velge å oppheve sperren ved å trykke enten SAMTYKKE eller AKUTT som grunnlag for denne overstyringen.  Overstyring av samtykke må brukslogges spesielt, og pasienten vil få melding om at noen har overstyrt deres sperrer. 
Er denne overstyrings-muligheten noe som bør inn i disse tillits-tokenene? 
(Isåfall betyr det at Kjernejournal portal må gjøre en token-exchange og legge til denne ekstra attributten i tokenet - ettersom det å trykke denne knappen er noe som skjer inne i KJ og ikke EPJ kan vite noe om.)

**Comment by AMJohnsen** [comment]
En annen tanke er om man også skulle ha fått:
- samtykke
- Unntatt fra samtykke
- akutt i tillegg til 
- Forhøyet Samtykke
- forhøyet akutt som @ChristinGorman skriver (de to sistnevnte er ved overstyring av en sperre).

**Comment by richardhus** [comment]
Pr. i dag er det kun mulig å overstyre sperre for å få tilgang til å se dokumentoversikten under journaldokumenter-fanen. Det er ingen muligheter for å overstyre sperringer som ligger i kildenes EPJ-systemer, og en slik funksjonalitet er neppe akseptabel for alle (noen?) kilder. Å kunne overstyre pasientens sperre lokalt i et journalsystem er hos noen kilder et privilegium som er gitt et ganske begrenset antall helsepersonell, så det skal nok veldig gode argumenter til for at disse er villige til å innvilge dette for eksternt helsepersonell. Det har vært diskutert om man kunne gi flere helsepersonell muligheten til å overstyre sperrer dersom en kollega også attesterer at behovet for å overstyre en sperre er reelt, men det dekker ikke alle scenarier det heller.

Husk også at en varsling om overstyring til pasienten er reaktivt, og da er det kanskje allerede for seint fordi skaden allerede har skjedd sett fra pasienten som har sperret sitt ståsted. Det er mennesker i Norge som opplever grov undertrykkelse, og de får ikke nødvendigvis spesielt mye nytte av at de får varsel om brutt sperring hvis de ikke kan forfølge dette likevel.

Det vil fortsatt være mulig å ta en telefon eller på annen måte be kildene om tilgang til sperret informasjon de måtte ha utenom denne elektroniske løsningen. Da er man sikker på at det gjøres en menneskelig vurdering hos kilden av om informasjonen som er sperret kan være relevant og nødvendig for det samarbeidende helsepersonellet og om det er sannsynliggjort at dette vil være i pasientens interesse å utlevere.

Sperrede dokumenter vises heller ikke i dokumentoversikten uavhengig av valg om å overstyre sperren på fanen for journaldokumenter eller ikke.

Siden kjernejournalportalen heller ikke viser hvilken kilde det finnes sperrede dokumenter ved, kan det være krevende for det oppsøkende helsepersonellet (konsumenten) å finne ut hvordan de skal få avklart om den sperrede informasjonen er relevant og nødvendig for deres behandling av pasienten. Fra helseforetakene under Helse Sør-Øst RHF er det tilgjengelig hvilket helseforetak som har hvor mange sperrede dokumenter i vårt svar til nasjonal XCA. Men det kan være problematisk å avsløre hvor det finnes sperrede dokumenter dersom det er fra kilder som tilbyr veldig få og spissede helsetjenester (rusinstitusjon for eksempel), for da kan slike metadata også avsløre hvilken type behandling pasienten har mottatt bare ved at det finnes dokumenter fra denne virksomheten, selv om det kun fremkommer at det er sperrede dokumenter der.

Husk at selv om det generelt er slik at pasientsikkerhet trumfer personvern, så er ikke det helt svart/hvitt, for et liv kan også bli ødelagt eller i noen tilfeller også tatt som følge av personvernbrudd. Riktig balanse mellom pasientsikkerhet og personvern er helt kritisk for å opprettholde tilliten blant oss innbyggere til norsk helsevesen, slik at vi ikke begynner å oppsøke alternative behandlere eller begrenser hva vi deler med våre behandlere fordi vi frykter at helsevesenet ikke ivaretar vårt behov for diskresjon og konfidensialitet på en tilstrekkelig måte.

Hvis sperringer knapt nok oppleves som "security through obscurity" får vi fort utfordringer...

Men vi trenger uansett at det formidles fra kjernejournal om tilgangen til dokumentkilder har bakgrunn i en overstyring av slike sperrer eller ikke, slik at det kan logges og vises i innbyggers tilgangslogg også på kildesiden. Jeg mente egentlig at vi mottar dette allerede i dag i attributtet purpose som vi får i SAML, jeg mener da vi har sett ulike verdier i det avhengig av om vi har overstyrt sperre inn i pasientens kjernejournal eller overstyrt sperre på journaldokumenter-fanen når vi testet dette for en god tid tilbake. Men hukommelsen kan godt være feil her altså, og jeg er usikker på hvor nøye vi har testet alle kombinasjoner av unntatt fra samtykkekrav, overstyring av sperring for hele pasientens kjernejournal eller bare sperring av journaldokumenter-fanen...

**Comment by mortsten** [comment]
I dag gis denne informasjonen i purpose of use. Kodeverket ( [https://terminology.hl7.org/ValueSet-v3-PurposeOfUse.html](url) )  som nå foreslås i informasjonsmodellen inneholder ikke samtykke men emergency og brake the glass

**Reply by troelde** [reply]
Er det ikke koden TREAT treatment som da benyttes?

**Comment by steinarnoem** [comment]
Feature team oppfatter saken slik at dagens løsning er god nok til å dekke behovet, og at saken derfor lukkes
